tests/testthat/helper.R

```diff
@@ -28,3 +28,40 @@ test_create_target_directory <- function(path = NULL,
     target_dir <- file.path(path, add_level)
   }
 }
+
+
+# custom helper function to extract column value from from `stress_test_arguments`
+# for a given `name`, converted to numeric
+get_st_arg_val_num <- function(arg, col, arg_tibble = stress_test_arguments) {
+
+  arg_val <- arg_tibble %>%
+    dplyr::filter(name == !!arg) %>%
+    dplyr::pull(!!col)
+
+  return(as.numeric(arg_val))
+}
+
+#' Get a valid value of an argumetn of `run_stress_test()`
+#'
+#' @examples
+#' get_st_argument("term")
+#' # Same
+#' get_st_argument("term", "default")
+#'
+#' get_st_argument("term", "min")
+#'
+#' typeof(get_st_argument("company_exclusion"))
+#' typeof(get_st_argument("asset_type"))
+#' @noRd
```

Comment:
Here is an alternative implementation of the helper:

``` r
get_st_argument("term")
#> [1] 2
# Same
get_st_argument("term", "default")
#> [1] 2

get_st_argument("term", "min")
#> [1] 1

typeof(get_st_argument("company_exclusion"))
#> [1] "logical"
typeof(get_st_argument("asset_type"))
#> [1] "character"
```

<sup>Created on 2021-12-10 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
used yours now